NOTICE

```diff
@@ -1,29 +0,0 @@
-Copyright 2009-2011 Software Engineering Professionals, Inc.
-Copyright 2011-2012 Matt Burke
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-   http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-
-
-This software distribution contains the following libraries:
-
-NDesk.Options, Copyright (C) 2008 Novell (http://www.novell.com)
-http://www.ndesk.org/Options
-
-LibGit2Sharp, Copyright (c) 2011-2012 LibGit2Sharp contributors
-https://github.com/libgit2/libgit2sharp/
-
-libgit2, Copyright (C) 2009-2012 the libgit2 contributors
-http://libgit2.github.com/
-
-StructureMap, copyright 2004-2009 Jeremy D. Miller
-http://structuremap.sourceforge.net/
```

Comment:
Can the `NOTICES` file be added back in? The [Apache License refers to it explicitly](https://github.com/git-tfs/git-tfs/blob/d5923f7a1331faf9a8cdf618601015b8e67e9b56/LICENSE#L107-L122).